integration/commands_administration_test.go

```diff
@@ -986,3 +986,75 @@ func TestCommandsAdministrationCurrentOp(t *testing.T) {
 	_, ok := must.NotFail(doc.Get("inprog")).(*types.Array)
 	assert.True(t, ok)
 }
+
+func TestCommandsAdministrationListIndexes(t *testing.T) {
+	t.Parallel()
+
+	ctx, targetCollections, compatCollections := setup.SetupCompat(t)
+
+	for i := range targetCollections {
+		targetCollection := targetCollections[i]
+		compatCollection := compatCollections[i]
+
+		t.Run(targetCollection.Name(), func(t *testing.T) {
+			t.Parallel()
+
+			targetCur, targetErr := targetCollection.Indexes().List(ctx)
+			compatCur, compatErr := compatCollection.Indexes().List(ctx)
+
+			require.NoError(t, compatErr)
+			assert.Equal(t, compatErr, targetErr)
+
+			targetRes := FetchAll(t, ctx, targetCur)
+			compatRes := FetchAll(t, ctx, compatCur)
+
+			// TODO Use simple assert.Equal after https://github.com/FerretDB/FerretDB/issues/1384
+			// assert.Equal(t, compatRes, targetRes)
+			assert.Empty(t, targetRes)
+			assert.NotEmpty(t, compatRes)
+		})
+	}
+}
+
+// TestCommandsAdministrationRunCommandListIndexes tests the behavior when listIndexes is called through RunCommand.
+// It's handy to use it to test the correctness of errors.
```

Comment:
I don't think we should test through `RunCommand` in general: we want compatibility with apps through drivers, not with specs/docs via `RunCommand`. There are some exceptions like `drop` that hide some errors, but they are exceptions. There is no need to over-test simple commands like `listIndexes`